buildspecs/aix_ppc-64.spec

```diff
@@ -221,6 +221,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="module_zlib" value="true"/>
 		<flag id="opt_annotations" value="true"/>
 		<flag id="opt_bigInteger" value="true"/>
+		<flag id="opt_criuSupport" value="false"/>
```

Comment:
Setting the variable to false in every spec is not required. This change was removed from a previous PR.

Reply:
nm, I see you comment now.

Reply:
I think the sed script relies on it being there for enablement.

https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/OpenJ9.gmk#L205

I wasnt able to get it to work without it.

Reply:
Turns out if the flag is defined in core.feature it doesnt need to be defined in each spec, so I've added it there.